CODING_STYLE.md

```diff
@@ -102,11 +102,10 @@ our test framework and the guidelines on how to write and run unit tests.
 ### Commit messages
 All commit lines (entered when you run `git commit`) must follow the common
 conventions for git commit messages:
-- The first line is a short summary, no longer than **50 characters,** starting
-  with an area name and then a colon.  There should be no period after
-  the short summary.
+- The first line is a short summary starting with an area name and then a colon.
```

Comment:
no longer than **72 characters,**

Reply:
50 characters hard limit is gone, but the rule is still there.